.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "res/logo"]
+	path = res/logo
+	url = git@github.com:tarekdj/Pickle-Logo.git
```

Comment:
you should use the http or git URL to allow installing the logo in an anonymous way (see the Travis failure)